wca-regulations.md

```diff
@@ -413,11 +413,11 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A7a1) If the judge finds that the puzzle is solved, they call "OKAY".
         - A7a2) If the judge assigns any penalties, they call "PENALTY".
         - A7a3) If the result is DNF, the judge calls "DNF".
-    - A7b) The judge records the result on a score sheet.
+    - A7b) The judge records the result on a score sheet and signs next to the result.
```

Comment:
Are we talking about the same thing? I was talking about changing the design of the score sheets.

I am sure this will confuse some people: judges will sign in the wrong column or competitors will ask why judges signed in the wrong column.
I think it's bad to change it right before the largest and the third (?) largest competition ever, as it will only generate more issues and maybe even incidents.